streams/src/test/java/org/apache/kafka/streams/integration/WordCountIntegrationTest.java

```diff
@@ -139,6 +141,16 @@ public String apply(final String key, final String value) {
         consumerConfig.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, LongDeserializer.class);
         List<KeyValue<String, Long>> actualWordCounts = IntegrationTestUtils.waitUntilMinKeyValueRecordsReceived(consumerConfig,
             DEFAULT_OUTPUT_TOPIC, expectedWordCounts.size());
+        Collections.sort(actualWordCounts, new Comparator<KeyValue<String, Long>>() {
```

Comment:
Why do we need to sort? It would be good if values sent match the ones received without sorting.